lib/dry/system/loader.rb

```diff
@@ -23,61 +23,53 @@ module System
     #
     # @api public
     class Loader
-      # @!attribute [r] component
-      #   @return [Dry::System::Component] Component to be loaded
-      #   @api private
-      attr_reader :component
+      class << self
+        # Requires the component's source file
+        #
+        # @api public
+        def require!(component)
+          require(component.path) if component.file_exists?
```

Comment:
@timriley is this OK that it's gonna silently skip require when the component's file doesn't exist? Maybe it should raise an error in such case?

Reply:
ps. I know it used to work like that but it could be an opportunity to improve it

Reply:
@solnic I'm happy to reconsider this! Perhaps in a follow-up PR?

Reply:
@timriley sounds good! It caught my attention because I remember that it can be really daunting to track down a configuration issue when dry-system is silently doing nothing. I'm sure you know what I' talking about. This also reminds me we need a debug mode with lots of logging. I'm happy to take care of this!